### DISCUSSION TITLE
"katex" issues

### DISCUSSION THREAD

**Discussion by FuadEfendi** [hugomods/site#1112] [Q&A] [answered]
Hi Razon,

This simple test shows $$ intstead of formulas; I have some pages like that, they show $$ and I don't know why:

```markdown
---
type: docs
---

{{< katex >}}

\text{Coupon per period} = \frac{\text{Annual Coupon Rate}}{2} \times \text{Face Value}
{{< /katex >}}

{{< katex >}}

\text{Coupon per period} = \frac{9	ext{%}}{2} \times \$100 = 4.5	ext{	ext{%}} \times \$100 = \$4.50

{{< /katex >}}

```

For example this page:
https://securitiesexamprep.ca/3.-investment-products/7.-fixed-income-securities-pricing-and-trading/7.2-calculating-price-and-yield-of-bond/7.2.1-discount-rate/

**Comment by FuadEfendi** [comment]
Interestingly, I just realized that this syntax works perfectly:
```
$$
P = \sum_{t=1}^{T} \frac{C}{(1+r)^t} + \frac{F}{(1+r)^T}
$$

Where:

- \\(P\\) = price of the bond
- \\(C\\) = coupon payment
- \\(F\\) = face value of the bond
- \\(T\\) = number of periods to maturity
- \\(r\\) = YTM
```

**Reply by FuadEfendi** [reply]
I think when I have {{< katex >}} on a page, it activates JavaScript, and after that I can use even standard $$ (display) or \(...)\ (inline)

$$ is shown probably because error in formula

**Comment by razonyang** [comment, marked as answer]
I'm not fimilar to KaTeX formula, but your code seems wrong.

![image](https://github.com/hugomods/site/assets/17720932/2fc8dc64-75a6-4412-ae18-07b080449b04)

> $$ is shown probably because error in formula

Yep, the KaTeX module just wrap the formula with `$$` (you can find it from source code, CRTL+U), the rest part is handled by KaTeX JavaScript.
You should check if the formula is correct, if this issue still present with correct formula, please provide a sample.
